src/main/java/com/stream_pi/client/window/dashboard/actiongridpane/ActionBox.java

```diff
@@ -482,7 +484,7 @@ public void setDisplayTextFontColourAndSize(String colour)
         }
     }
     
-    public void playActionAnimation() throws SevereException
+    public void playActionAnimation() throws MinorException, SevereException
```

Comment:
Throwing minorexception here, only to handle it where its called. Looks redundant. Remove Exceptions from here, handle them inside this method itself